PARTNERS.md

```diff
@@ -0,0 +1,49 @@
+# OADP Partner Information
+
+## Important Announcement: Version Support Changes
+Starting in 2024, OADP will implement a streamlined version support policy. Red Hat will support only one version of OADP per OpenShift version to ensure better stability and maintainability.
```

Comment:
```suggestion
## Important Announcement: Version Support Changes
Starting in 2025, OADP will implement a streamlined version support policy. Red Hat will support only one version of OADP per OpenShift version to ensure better stability and maintainability.
```

Given the current time of year and this PR's recent creation, should this be 2025?